src/jest.config.js

```diff
@@ -17,4 +18,21 @@ module.exports = {
   modulePathIgnorePatterns: ['<rootDir>/dist/'],
   moduleDirectories: ['node_modules', 'src'],
   rootDir: './',
+  transform: {
+    '^.+\\.tsx?$': [
+      'ts-jest',
+      {
+        tsconfig: 'tsconfig.json',
+        isolatedModules: true,
+      },
+    ],
+  },
+  transformIgnorePatterns: ['/node_modules/(?!ethereum-cryptography)'],
+  moduleNameMapper: {
+    '\\.(css|less|sass|scss)$': 'identity-obj-proxy',
```

Comment:
don't seems to be necessary as well

Reply:
same as the other:

```
I broke down the tests into three separate configurations:

- use different timeouts when running just unit vs (integration or combined)
- use different parallelization when running just unit vs (integration or combined)
- have separate coverage outputs when running individual(unit or integration) or combined

In the end, the CI will consider the combined approach as the total coverage, while you can still run them individually and analyze separately the results```

Reply:
Sorry, if you were referring to the ignore pattern precisely, it was necessary due to some internal imports from the cacti test tooling package :(